### DISCUSSION TITLE
How to open the PR with a specific browser?

### DISCUSSION THREAD

**Discussion by itsbetma** [jesseduffield/lazygit#3952] [Q&A] [answered]
I am using MacOS and don't want to set chrome as default browser because I just use chrome for work. Is there a possible way to have Safari as default browser on Mac and Chrome as the browser to be used on LazyGit?

**Comment by stefanhaller** [comment, marked as answer]
Try this in your lazygit config file:
```yaml
os:
    openLink: open -a "/Applications/Google Chrome.app" {{link}}
```

**Reply by itsbetma** [reply]
Thank you so much
